client/src/util/typedCrossfilter/bitArray.js

```diff
@@ -211,6 +211,19 @@ class BitArray {
     }
   }
 
+  // select range of indices on a dimension
```

Comment:
My bit manipulation isn't strong enough to properly review this code

Reply:
there are no significant changes to this code in quite some times.   I added a new interface in this PR, but the logic is identical.

Reply:
👍